pkg/authz/cedar.go

```diff
@@ -258,8 +259,17 @@ func (a *CedarAuthorizer) IsAuthorized(
 		entityMap = mergedEntities
 	}
 
+	// Debug logging for authorization
+	logger.Debugf("Cedar authorization check - Principal: %s, Action: %s, Resource: %s",
+		req.Principal, req.Action, req.Resource)
+	logger.Debugf("Cedar context: %+v", req.Context)
+
 	// Check authorization
 	decision, diagnostic := cedar.Authorize(a.policySet, entityMap, req)
+
+	// Log the decision
+	logger.Debugf("Cedar decision: %v, diagnostic: %+v", decision, diagnostic)
```

Comment:
This looks super handy. Should we add this to the context so we can retrieve it in the audit middleware?  It would be good to add it there.